File: pki/src/main/java/com/dreamteam/pki/dto/certificate_holder/response/CreateEntityResponse.java
package com.dreamteam.pki.dto.certificate_holder.response;

import lombok.Data;

@Data
public class CreateEntityResponse {
    private String email;
    private String password;
    private String commonName;
    private String country;
    private String state;
    private String locality;
    private String organization;
    private String organizationalUnit;
}

File: employee-management-app/src/app/pages/login/models/response/CreateEntityResponse.ts
import { CertificateHolderType } from "src/app/core/models/certificate-holder-type";

export class CreateEntityResponse {
  id: string;
  email: string;
  type: string;
  commonName: string;
  country: string;
  state: string;
  locality: string;
  organization: string;
  organizationalUnit: string;

  constructor(
      id: string,
      email: string,
      commonName: string,
      country: string,
      state: string,
      locality: string,
      organization: string,
      organizationalUnit: string
  ) {
      this.id = id;
      this.email = email;
      this.type = CertificateHolderType.ENTITY;
      this.commonName = commonName;
      this.country = country;
      this.state = state;
      this.locality = locality;
      this.organization = organization;
      this.organizationalUnit = organizationalUnit;
  }
}
